id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/manager/CredentialServiceManager.java

```diff
@@ -274,11 +273,21 @@ private void sendUINEventToIDA(String uin, LocalDateTime expiryTimestamp, String
 						.collect(Collectors.toList()));
 
 		if (vidInfoDtos != null) {
-			List<EventModel> idaEvents = vidInfoDtos.stream()
+			List<EventModel> vidEvents = vidInfoDtos.stream()
 					.flatMap(vidInfoDTO -> createIdaEventModel(eventType, expiryTimestamp,
 							vidInfoDTO.getTransactionLimit(), partnerIds, txnId, vidInfoDTO.getHashAttributes().get(IdRepoConstants.ID_HASH)))
 					.collect(Collectors.toList());
-			eventList.addAll(idaEvents);
+			eventList.addAll(vidEvents);
+		}
+
+		if (handles != null && !handles.isEmpty()) {
+			mosipLogger.debug(IdRepoSecurityManager.getUser(), this.getClass().getCanonicalName(), "sendUINEventToIDA",
+					"Number of handles identified >> " + handles.size());
```

Comment:
Update the status as "DELETE_REQUESTED" for all the handles linked to deactivated UIN

Reply:
Fixes done in https://github.com/mosip/id-repository/pull/964 PR.